browser/ui/views/frame/brave_browser_frame_mac.mm

```diff
@@ -56,3 +57,24 @@
   result->new_toggle_state = !model->empty() && model->GetActiveWebContents() &&
                              model->GetActiveWebContents()->IsAudioMuted();
 }
+
+bool BraveBrowserFrameMac::ExecuteCommand(
+    int32_t command,
+    WindowOpenDisposition window_open_disposition,
+    bool is_before_first_responder) {
+  /* is_before_first_responder tells whether or not the app/window was in focus
+     while the keyboard command was fired. In current method, it helps in
+     distinguishing the 'file -> close tab' (false, as toolbar was in focus)
+     command from 'ctrl + w' (true, as tab was in focus) command.*/
```

Comment:
Hey! I actually ran `npm run format`, it didn't change anything in comments. However, I've manually made some changes observing the comment on top of the class.

Reply:
Could you try adding the whole comment as a single line and then formatting?

```cpp
// is_before_first_responder tells whether or not the app/window was in focus while the keyboard command was fired. In current method, it helps in distinguishing the 'file -> close tab' (false, as toolbar was in focus) command from 'ctrl + w' (true, as tab was in focus) command.
```

I think Chromium normally uses a bunch of line comments rather than multi line comments, which might be tripping the formatter up

Reply:
> Could you try adding the whole comment as a single line and then formatting?
> 
> ```c++
> // is_before_first_responder tells whether or not the app/window was in focus while the keyboard command was fired. In current method, it helps in distinguishing the 'file -> close tab' (false, as toolbar was in focus) command from 'ctrl + w' (true, as tab was in focus) command.
> ```
> 
> I think Chromium normally uses a bunch of line comments rather than multi line comments, which might be tripping the formatter up
thanks for the suggestion @fallaciousreasoning! it worked.